### ISSUE TITLE
Issue running site-packages

### ISSUE THREAD

**Issue by bbidonb** [snakemake-workflows/rna-seq-star-deseq2#74]
Hello,
Thank you for useful workflow.
I think I have created the snakemake environment as described as well as I deployed your workflow.
But it gives me back error when running, supposedly due to unavailable packages.

```
(/CONDAS/users/bbidon/snakemake) [bbidon@gknwwd2 covid]$ snakemake 
Traceback (most recent call last):
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/cli.py", line 1933, in args_to_api
    dag_api = workflow_api.dag(
              ^^^^^^^^^^^^^^^^^
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/api.py", line 328, in dag
    return DAGApi(
           ^^^^^^^
  File "<string>", line 6, in __init__
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/api.py", line 438, in __post_init__
    self.workflow_api._workflow.dag_settings = self.dag_settings
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/api.py", line 385, in _workflow
    workflow.include(
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/workflow.py", line 1386, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "/SCRATCH-BIRD/users/bbidon/ntu/covid/workflow/Snakefile", line 24, in <module>
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/workflow.py", line 2125, in decorate
    module.use_rules(
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/modules.py", line 104, in use_rules
    self.workflow.include(snakefile, overwrite_default_target=True)
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/workflow.py", line 1386, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "https://raw.githubusercontent.com/snakemake-workflows/rna-seq-star-deseq2/v2.1.0/workflow/Snakefile", line 31, in <module>
    RerunTrigger,
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/workflow.py", line 1386, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "https://raw.githubusercontent.com/snakemake-workflows/rna-seq-star-deseq2/v2.1.0/workflow/rules/common.smk", line 7, in <module>
    import hashlib
          ^^^^^^^^^
  File "/CONDAS/users/bbidon/snakemake/lib/python3.12/site-packages/snakemake/remote/__init__.py", line 8, in __init__
    raise NotImplementedError(
NotImplementedError: Remote providers have been replaced by Snakemake storage plugins. Please use the corresponding storage plugin instead (snakemake-storage-plugin-*).
(/CONDAS/users/bbidon/snakemake) [bbidon@gknwwd2 covid]$ ls -lsha
```

Do you have any idea about the origin of the problem.
Thank you,
Baptiste

Edit by @dlaehnemann: turn into multi-line column

**Comment by dlaehnemann** [comment]
Can you try downgrading the `python` in your `snakemake` environment to `3.11`? Python does not follow semantic versioning and a lot of things broke in the step from `3.11` to `3.12`, so this might fix it. Simply do (with your `snakemake` conda environment activated):
```
conda install python=3.11
```

**Comment by bbidonb** [comment]
Hi,
I downgraded python within the conda environment with the above mentioned command.
`
(/CONDAS/users/bbidon/snakemake) [bbidon@6t4jb5j covid]$ python --version
Python 3.11.8
`
I still got the same error.

**Comment by dlaehnemann** [comment]
I was on the wrong track initially, but have now looked into it. This should be fixed with #76 and if you upgrade to the latest release of the workflow, this should work (`tag="v2.1.1"`).
